data/campaigns/World_Conquest/gui/invest_dialog.cfg

```diff
@@ -324,7 +324,7 @@
                         [column]
                             horizontal_alignment = "right"
                             [button]
-                                label = "Ok"
+                                label = _ "Ok"
```

Comment:
This should probably be a core string from a different textdomain. I'm not quite sure where that would be found though.

Reply:
Revert.